1. Basic Web Development/server/src/Model/Permissions.ts

```diff
@@ -0,0 +1,19 @@
+import mongoose from "mongoose";
+
+export interface PermissionDocument extends mongoose.Document {
+    type: string;
```

Comment:
You dropped the type enum in order to implement CRUD for permission types but that doesn't yet seem to be done.

Reply:
Categories are now generated dynamically and permissions get their type from the id of the category the user checked when creating the permission. This was done in order to have binding between between categories and permissions. The logic [here](https://github.com/dochiasebastian/workshops/pull/10/files#diff-e7690af629a262ea8d7f5c1d8cb4a1e299cb564773268dc32c08f6edc3f8f927R340-R347) changed and is no longer a switch between predefined types.